src/dummy/d2d1.h

```diff
@@ -176,24 +178,51 @@ enum dummy_D2D1_LINE_JOIN_tag {
   dummy_D2D1_LINE_JOIN_ROUND = 2
 };
 
+typedef enum dummy_D2D1_GAMMA_tag dummy_D2D1_GAMMA;
+enum dummy_D2D1_GAMMA_tag {
+  dummy_D2D1_GAMMA_2_2 = 0,
+  dummy_D2D1_GAMMA_1_0 = 1,
+  dummy_D2D1_GAMMA_FORCE_DWORD = 2
+};
+
+typedef enum dummy_D2D1_EXTEND_MODE_tag dummy_D2D1_EXTEND_MODE;
+enum dummy_D2D1_EXTEND_MODE_tag {
+  D2D1_EXTEND_MODE_CLAMP = 0,
+  D2D1_EXTEND_MODE_WRAP = 1,
+  D2D1_EXTEND_MODE_MIRROR = 2,
+  D2D1_EXTEND_MODE_FORCE_DWORD = 3
+};
```

Comment:
Please add the prefix "dummy_" to the enum members. Enum members are global identifiers in C.